onnxruntime/core/graph/schema_registry.cc

```diff
@@ -176,6 +174,34 @@ DomainToVersionMap SchemaRegistryManager::GetLatestOpsetVersions(bool is_onnx_on
       }
     }
   }
+}
+
+DomainToVersionMap SchemaRegistryManager::GetLastReleasedOpsetVersions(bool is_onnx_only) const {
```

Comment:
Can we save the copy of the map in the return path?

Reply:
Right it is written this way to maintain consistency with the other Get API. The calling code calls one of the 2 apis based on the value of the env variable "ALLOW_RELEASED_ONNX_OPSET_ONLY"